### ISSUE TITLE
JAWS ignores accessible window title provided by Chrome and doesn't announce the window when it's being opened

### ISSUE THREAD

**Issue by mshoho** [FreedomScientific/standards-support#323]
### Summary

Two parts of the issue.

First one is that Chrome provides accessible window title for the screen readers which might be different from the visible title. JAWS seems to ignore that title. One of the places it's being used is an incognito window — the visible title is the same to a normal window title, but the screen reader title has «Incognito» in it. NVDA announces accessible window title when I open a new tab (or when I switch to the incognito window from another window), it reads something like «New tab Google Chrome, Incognito...». When I press Modifier+T to read the current window title, JAWS reads the visible title only.

The second part of the issue is when a new window opens (say, I press Ctrl+N or Ctrl+T), JAWS says «Ctrl+N, Address and search bar edit, type in text» or «Ctrl+T», and doesn't say a newly opened window title.

### Expected result

I expect JAWS to announce the accessible title provided by the browser. And I would also expect a window title announcements when a new window opens (or when an already opened window gets activated).

### Actual result

I hear a visible title only when I press Modifier+T. And nothing when I switch to a new window.

#### JAWS version and build number

JAWS 2019.1909.28 ILM.
 
#### Operating System and version

Windows 10.

#### Browser and version:

Chrome 77.0.3865.90 (Official Build) (64-bit)

**Comment by stevefaulkner** [comment]
@mshoho  Closing this, please feel free to re-open if this still occurs with Latest (JAWS 2024).
